lib/i18n/tasks/command/options/common.rb

```diff
@@ -26,7 +26,7 @@ module Common
         arg :value,
             '-v',
             '--value VALUE',
-            t('i18n_tasks.cmd.args.desc.value')
+            t('i18n_tasks.cmd.args.desc.value', dummy: 'value') # Dummy value is workaround for https://github.com/ruby-i18n/i18n/issues/689
```

Comment:
is the workaround still necessary, since ruby-i18n/i18n#689 has been merged and a new version released?